src/main/java/uk/gov/hmcts/reform/hmc/config/MessageReceiverConfiguration.java

```diff
@@ -0,0 +1,98 @@
+package uk.gov.hmcts.reform.hmc.config;
+
+import com.azure.core.amqp.AmqpRetryOptions;
+import com.azure.core.util.BinaryData;
+import com.azure.messaging.servicebus.ServiceBusClientBuilder;
+import com.azure.messaging.servicebus.ServiceBusReceiverClient;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestClientException;
+import uk.gov.hmcts.reform.hmc.ApplicationParams;
+import uk.gov.hmcts.reform.hmc.client.futurehearing.ActiveDirectoryApiClient;
+import uk.gov.hmcts.reform.hmc.client.futurehearing.HearingManagementInterfaceApiClient;
+
+import java.time.Duration;
+import javax.annotation.PostConstruct;
+
+@Slf4j
+@Component
+public class MessageReceiverConfiguration implements Runnable {
+
+    private final ApplicationParams applicationParams;
+    private final ActiveDirectoryApiClient activeDirectoryApiClient;
+    private final HearingManagementInterfaceApiClient hmiClient;
+    private static final String MESSAGE_TYPE = "message_type";
+    private static final ObjectMapper OBJECT_MAPPER = new Jackson2ObjectMapperBuilder()
+        .modules(new Jdk8Module())
+        .build();
+
+    public MessageReceiverConfiguration(ApplicationParams applicationParams,
+                                        ActiveDirectoryApiClient activeDirectoryApiClient,
+                                        HearingManagementInterfaceApiClient hmiClient) {
+        this.activeDirectoryApiClient = activeDirectoryApiClient;
+        this.applicationParams = applicationParams;
+        this.hmiClient = hmiClient;
+    }
+
+    @Override
+    @SuppressWarnings("squid:S2189")
+    @PostConstruct
+    public void run() {
+        log.info("Creating Session receiver");
```

Comment:
```suggestion
        log.info("Creating service bus receiver client");
```
It's no longer a session receiver